ecommerce/extensions/api/v2/views/webhooks.py

```diff
@@ -34,7 +34,7 @@ class StripeWebhooksView(APIView):
     @csrf_exempt
     def post(self, request):
         payload = request.body
-        sig_header = request.META['HTTP_STRIPE_SIGNATURE']
+        sig_header = request.headers['stripe-signature']
```

Comment:
We can skip this change for Django 42 support as well.
```suggestion
        sig_header = request.META['HTTP_STRIPE_SIGNATURE']
```